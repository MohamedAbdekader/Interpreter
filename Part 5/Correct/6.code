program
	add(a, b) begin
		a = new a+b;
		define n;
		b = define n;
	endfunc
begin
	define x;
	x = new 1;
	define y;
	y = new 1+x;
	begin add(x, y);
	output x;
end