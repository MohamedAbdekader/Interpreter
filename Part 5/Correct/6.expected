gc:1
gc:2
gc:3
gc:2
gc:1
3
gc:0
